vcd/provider.go

```diff
@@ -94,6 +95,7 @@ var globalResourceMap = map[string]*schema.Resource{
 	"vcd_vapp_nat_rules":      resourceVcdVappNetworkNatRules(),      // 2.9
 	"vcd_vapp_static_routing": resourceVcdVappNetworkStaticRouting(), // 2.9
 	"vcd_vm_affinity_rule":    resourceVcdVmAffinityRule(),           // 2.9
+	"vcd_access_control_vapp": resourceVcdAccessControlVapp(),        // 2.10
```

Comment:
Adding here reference that we have discussion regarding resource name `vcd_access_control_vapp` or `vcd_vapp_access_control`

Reply:
Let's consider that:
* we will implement access control also for catalog and VDC;
* all access control operations use the same data structure and a very similar API call;
* the result of the operation is not stored in the resource internal structure.

That said, let's decide: which combination looks more appealing?
1. suffix
    * `vcd_access_control_vapp`
    * `vcd_access_control_catalog`
    * `vcd_access_control_vdc`

2. prefix
    * `vcd_vapp_access_control`
    * `vcd_catalog_access_control`
    * `vcd_vdc_access_control`

Reply:
@lvirbalas @Didainius

Reply:
IMO - 2. It follows the "namespace".

Reply:
As a user, I stumbled a little here. Funny. 
I intentionally wrote it like 2.prefix, and named it "vappacl". Similar to vcd_vapp_org_network:
```
resource "vcd_vapp" "vapp" {
resource "vcd_access_control_vapp" "vappacl" {
resource "vcd_vapp_org_network" "vappnetattach" {
resource "vcd_vapp_vm" "instance" {
```
If it follows the GUI: You create a vapp, then set network in the dialog. You create a vapp, then set acl in the dialog...

So as a user for usability I prefer `vcd_vapp_access_control`

Reply:
I'm on a fence on this one, but appreciate the 2nd option `vcd_vapp_access_control` slightly more.

Reply:
I like more `vcd_vapp_access_control` more.

Reply:
renamed to `vcd_vapp_access_control`

Reply:
From my point of view, I only took vcd_vapp_access_control as an example for all 3 cases. Now it is mixed?
```
vcd_vapp_access_control
vcd_access_control_catalog
vcd_access_control_vdc

```

Reply:
`vcd_access_control_catalog` and `vcd_access_control_vdc` don't exist yet. When they do, they will be named appropriately